piyushgarg-dev/review-app
# fix: CreateProject-Logout-Issue resolved

## What does this PR do?

Fixes #49

##Issue Summary

Even after logging out the Modal opens when the "Create Project" button is clicked and is available to create a new project which will not be created even if we create a new project.

##Solution
When you are logged out and try to create a new project you will see a Modal that asks for login and a Login Button which redirects to the Login Page.



https://github.com/piyushgarg-dev/review-app/assets/77155890/592b98a3-9d41-470c-aad4-8bd196205b13






## Type of change




## How should this be tested?



- Login to your account.
- Now log out from your account and try to create a new project.
- the modal will be opened to create a new project
- Now you will see a Modal that asks for a login and a Login Button that redirects to the Login Page.
